packages/nodes-base/nodes/Microsoft/Sql/MicrosoftSql.node.ts

```diff
@@ -269,7 +269,7 @@ export class MicrosoftSql implements INodeType {
 						);
 					}
 					const results = await executeSqlQueryAndPrepareResults(pool, rawQuery, i);
-					returnData.push(...results);
+					returnData.concat(results);
```

Comment:
`concat` doesn't mutate the existing array, and instead returns a new array.

```suggestion
					returnData = returnData.concat(results);
```

Reply:
🤦🏻 Of course, Thanks for checking @netroy

Anything else you think we can do to improve this? I will get a test container set up on Monday to make sure this is all good.

Reply:
this looks good